examples/gno.land/p/demo/bf/bf.gno

```diff
@@ -0,0 +1,66 @@
+package bf
+
+import (
+	"strings"
+)
+
+func Execute(code string) string {
+	var (
+		memory  = make([]byte, 30000) // memory tape
+		pointer = 0                   // initial memory pointer
+		buf     strings.Builder
+	)
+
+	// Loop through each character in the code
+	for i := 0; i < len(code); i++ {
+		switch code[i] {
+		case '>':
+			// Increment memory pointer
+			pointer++
+		case '<':
+			// Decrement memory pointer
+			pointer--
+		case '+':
+			// Increment the byte at the memory pointer
+			memory[pointer]++
+		case '-':
+			// Decrement the byte at the memory pointer
+			memory[pointer]--
+		case '.':
+			// Output the byte at the memory pointer
+			buf.WriteByte(memory[pointer])
+		case ',':
+			// Input a byte and store it in the memory
+			panic("unsupported")
+			// fmt.Scan(&memory[pointer])
+		case '[':
+			// Jump forward past the matching ']' if the byte at the memory pointer is zero
+			if memory[pointer] == 0 {
+				braceCount := 1
+				for braceCount > 0 {
+					i++
+					if code[i] == '[' {
+						braceCount++
+					} else if code[i] == ']' {
+						braceCount--
+					}
+				}
+			}
```

Comment:
we could make this use a stack of open brackets, popped each time we go out of a loop, probably better for memory/efficiency... but if you don't want to do it it's fine :)

Reply:
let's keep this for when we'll have better benchmarking tools as a good example of runtime and gas optimization.